L1Trigger/Phase2L1GMT/src/L1TPhase2GMTBarrelStubProcessor.cc

```diff
@@ -136,6 +161,7 @@ l1t::MuonStubCollection L1TPhase2GMTBarrelStubProcessor::makeStubs(const L1Phase
         }
       }
     }
+    std::cout << os.str() << std::endl;
   }
```

Comment:
Since these print out are used for emulator firmware comparison, we prefer to use the print out as it is easier for us. However, the print out are controlled by the verbose level, which should be off by default.